### DISCUSSION TITLE
Stopped Working today tried several reboots still full log errors

### DISCUSSION THREAD

**Discussion by thorrrr** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#467] [Q&A]
I have tried rebooting , but get several errors was working until yesterday for months and i have not touched this integration for months

https://dpaste.org/W39oq

**Comment by PiotrMachowski** [comment]
It looks like a problem with your network. Do vacuum controls work in HA?

**Comment by thorrrr** [comment]
Yes works fine from HA with commands but the map is very laboured like 5 seconds behind

**Reply by PiotrMachowski** [reply]
Sooooo it works?

**Comment by thorrrr** [comment]
This error originated from a custom integration.

Logger: custom_components.xiaomi_cloud_map_extractor.camera
Source: custom_components/xiaomi_cloud_map_extractor/camera.py:278
Integration: xiaomi_cloud_map_extractor (documentation, issues)
First occurred: 09:22:09 (1 occurrences)
Last logged: 09:22:09

Unable to retrieve map data

**Reply by PiotrMachowski** [reply]
Please respond in the same thread.

Does the map show up despite these errors?

**Comment by thorrrr** [comment]
Yes i can see the map

**Reply by PiotrMachowski** [reply]
In this case you can safely ignore these errors. Map is refreshed every 5s anyway, so a bit of lagging behind is expected.
